docs/advanced/storagenetwork.md

```diff
@@ -384,3 +384,19 @@ The storage network may malfunction because of issues with the external network,
 ### Start VM Manually
 
 After verifying the configuration, users could start VM manually on demand.
+
+### Best Practices
+
+- When allocating IP Range for storage network using the below formula,
+  Required Number of IPs = (Number of Nodes * 2) + (Number of Disks * 2) + Number of Images to Download/Upload
+  Consider allocating sufficient ip addresses keeping in mind the future needs of the cluster.
+  When new nodes are added to the cluster or more disks added to the node after storage network configuration and if required number of IPs becomes
+  greater than allocated, then it results in Longhorn pods (`instance-manager`, and `backing-image-manager`) not in running state.
+
+  Storage network has to be reconfigured with sufficient IP Range to solve this.
+
+- Users are allowed to configure storage networks using mgmt cluster, but it is not the best or recommended configuration as same physical nics are
+  shared by both storage network and other traffic over mgmt network.
+
+- Consider using storage network on mgmt cluster only if you could segregate traffic using vlans effectively on same physical link or the cluster has some NIC
```

Comment:
Maybe help the reader to understand what is the impact of running the storage network on the `mgmt` network. 

Suggestion:
```
Configuring the storage network on the managment (`mgmt`) network will have a negative impact on the control plane network performance, due to resource and bandwidth contention. Use this option only if you could efficiently segregate the two types of traffic.
```